src/coreclr/src/jit/importer.cpp

```diff
@@ -7353,7 +7355,8 @@ GenTree* Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolvedT
 
                     var_types handleTyp = (var_types)(varTypeIsGC(lclTyp) ? TYP_BYREF : TYP_I_IMPL);
                     op1                 = gtNewOperNode(GT_IND, handleTyp, op1);
-                    op1->gtFlags |= GTF_IND_INVARIANT | GTF_IND_NONFAULTING;
+                    op1->gtFlags |= GTF_IND_NONFAULTING;
+                    noway_assert(!"Unreachable on CoreCLR");
```

Comment:
If this is unreachable why not just remove it all together? We have already broken the ability to port the jit back to desktop.

Reply:
I don't think that this code path is only good for the desktop.
Instead it seems that the CoreCLR VM doesn't use this code path

Reply:
I will add an assert and remove the unreachable block, 
I found another section of code that asserted for the unreachable case.